### PR TITLE
Fedora perl package needed by podman docs checking

### PR DESCRIPTION
**Pull request by cevich**
See https://github.com/containers/podman/pull/8297

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by cevich** [comment]
@edsantiago reminder: An approving github review will cause this to merge (assuming build/test is successful)

File: cache_images/fedora_packaging.sh [hunk 1]
```diff
@@ -94,6 +94,7 @@ INSTALL_PACKAGES=(\
     openssl-devel
     ostree-devel
     pandoc
+    perl-FindBin
```
Comment (edsantiago) [inline cache_images/fedora_packaging.sh:97]: This is probably going to fail on pre-f33

Comment (cevich) [reply cache_images/fedora_packaging.sh:97]: good catch

**Review by edsantiago — APPROVED** [review]
Tested in https://github.com/containers/podman/pull/8306 ; confirmed working (i.e. the Validate task [ran xref-helpmsgs-manpages and passed](https://cirrus-ci.com/task/4712546554871808?command=main#L67))

**Comment by cevich** [comment]
Thanks @edsantiago for confirming, I appreciate it.

**Pull request merged**